Add support for virtual input

JaikrishnaNemallapudi opened the pull request:
Sending input events via spice protocol is disabled.
All the input events are sent to lg-input manager.

Signed-off-by: Jaikrishna Nemallapudi <nemallapudi.jaikrishna@intel.com>
Signed-off-by: Mallikarjun Chegaraddi, Raju <raju.mallikarjun.chegaraddi@intel.com>
Signed-off-by: RAJANI RANJAN <rajaniranjan@intel.com>

rajucm2007 commented on client/spice/src/spice.c:
```diff
@@ -1664,4 +1664,4 @@ bool spice_clipboard_data(SpiceDataType type, uint8_t * data, size_t size)
 
   free(buffer);
   return true;
-}
```
undo changes in this file.

rajucm2007 reviewed (commented):
please remove changes from spice.c

JaikrishnaNemallapudi committed:
Add support for virtual input

Sending input events via spice protocal is disabled.
All the input events are sent to lg-input manager.

Signed-off-by: Jaikrishna Nemallapudi <nemallapudi.jaikrishna@intel.com>
Signed-off-by: Mallikarjun Chegaraddi, Raju <raju.mallikarjun.chegaraddi@intel.com>
Signed-off-by: RAJANI RANJAN <rajaniranjan@intel.com>
